samples/main.go

```diff
@@ -449,6 +449,34 @@ func main() {
 			}
 			fmt.Printf("\n\n")
 		} else if choiceN == 18 {
+			fmt.Println("You selected list VPC volumes")
+			tags := map[string]string{}
+			volName := ""
+			zoneName := ""
+			fmt.Printf("Please enter ZONE Name to filter volumes(Optional): ")
+			_, er11 = fmt.Scanf("%s", &zoneName)
+			if zoneName != "" {
+				tags["zone.name"] = zoneName
+			}
+			fmt.Printf("Please enter volume Name to filter volumes(Optional): ")
+			_, er11 = fmt.Scanf("%s", &volName)
+			if volName != "" {
+				tags["name"] = volName
+			}
+
+			start := ""
+			var limit int
+			fmt.Printf("Please enter max number of volume entries per page to be returned(Optional: ")
+			_, er11 = fmt.Scanf("%d", &limit)
+			volumeobj1, er11 := sess.ListVolumes(limit, start, tags)
+			if er11 == nil {
```

Comment:
May be you can try printing the list.

Reply:
Already printing volume list in next line no. 473.
Also we are printing ListVolume response from API in provider/list_volumes.go file
sample output:
`{"level":"info","ts":"2020-03-17T16:26:38.023+0530","caller":"provider/list_volumes.go:49","msg":"Successfully retrieved volumes list from VPC backend","RequestID":"8976720f-b933-4740-8078-e7028d63ca63","VolumesList":{"first":{"href":"https://eu-gb.iaas.cloud.ibm.com/v1/volumes?start=16d1c5a9-d7ff-428e-b019-d61fcf97e079\u0026limit=2\u0026zone.name=eu-gb-1"},"next":{"href":"https://eu-gb.iaas.cloud.ibm.com/v1/volumes?start=d888bbb9-b474-4be2-8de2-f7f8c332fa2e\u0026limit=2\u0026zone.name=eu-gb-1"},"volumes":[{"href":"https://eu-gb.iaas.cloud.ibm.com/v1/volumes/16d1c5a9-d7ff-428e-b019-d61fcf97e079","id":"16d1c5a9-d7ff-428e-b019-d61fcf97e079","name":"poster-nutmeg-sympathy-data-undercut","capacity":100,"iops":3000,"resource_group":{"href":"https://resource-controller.cloud.ibm.com/v2/resource_groups/efd16c8eac35496e801cf46c19dd03d5","id":"efd16c8eac35496e801cf46c19dd03d5"},"profile":{"href":"https://eu-gb.iaas.cloud.ibm.com/v1/volume/profiles/general-purpose","name":"general-purpose"},"created_at":"2019-09-02T16:47:57Z","status":"available","volume_attachments":[{"id":"840db15c-9c88-4118-b978-1021b4fa250b","href":"https://eu-gb.iaas.cloud.ibm.com/v1/instances/8052bf7a-252b-47f8-bc59-ede160e4f456/volume_attachments/840db15c-9c88-4118-b978-1021b4fa250b","name":"attachment-16d1c5a9-d7ff-428e-b019-d61fcf97e079","type":"boot","delete_volume_on_instance_delete":true}],"zone":{"name":"eu-gb-1","href":"https://eu-gb.iaas.cloud.ibm.com/v1/regions/eu-gb/zones/eu-gb-1"},"crn":"crn:v1:bluemix:public:is:eu-gb-1:a/e242f140687cd68a8e037b26680e0f04::volume:16d1c5a9-d7ff-428e-b019-d61fcf97e079"},{"href":"https://eu-gb.iaas.cloud.ibm.com/v1/volumes/3e898aa7-ac71-4323-952d-a8d741c65a68","id":"3e898aa7-ac71-4323-952d-a8d741c65a68","name":"riaas_vol1","capacity":20,"iops":3000,"resource_group":{"href":"https://resource-controller.cloud.ibm.com/v2/resource_groups/efd16c8eac35496e801cf46c19dd03d5","id":"efd16c8eac35496e801cf46c19dd03d5"},"profile":{"href":"https://eu-gb.iaas.cloud.ibm.com/v1/volume/profiles/general-purpose","name":"general-purpose"},"created_at":"2019-09-02T16:53:41Z","status":"available","volume_attachments":[],"zone":{"name":"eu-gb-1","href":"https://eu-gb.iaas.cloud.ibm.com/v1/regions/eu-gb/zones/eu-gb-1"},"crn":"crn:v1:bluemix:public:is:eu-gb-1:a/e242f140687cd68a8e037b26680e0f04::volume:3e898aa7-ac71-4323-952d-a8d741c65a68"}],"limit":2}}
{"level":"info","ts":"2020-03-17T16:26:38.023+0530","caller":"metrics/metrics.go:65","msg":"Time to complete","RequestID":"8976720f-b933-4740-8078-e7028d63ca63","ListVolumes":1.694961287}
{"level":"info","ts":"2020-03-17T16:26:38.023+0530","caller":"provider/list_volumes.go:67","msg":"Exit ListVolumes","RequestID":"8976720f-b933-4740-8078-e7028d63ca63","start":"","filters":{"zone.name":"eu-gb-1"}}
{"level":"info","ts":"2020-03-17T16:26:38.023+0530","caller":"samples/main.go:473","msg":"Successfully got volumes list================>","RequestID":"8976720f-b933-4740-8078-e7028d63ca63","VolumesList":{"next":"d888bbb9-b474-4be2-8de2-f7f8c332fa2e","volumes":[{"volumeID":"16d1c5a9-d7ff-428e-b019-d61fcf97e079","provider":"VPC","volumeType":"vpc-block","capacity":100,"iops":"3000","tier":null,"az":"eu-gb-1","creationTime":"2019-09-02T16:47:57Z","crn":"crn:v1:bluemix:public:is:eu-gb-1:a/e242f140687cd68a8e037b26680e0f04::volume:16d1c5a9-d7ff-428e-b019-d61fcf97e079"},{"volumeID":"3e898aa7-ac71-4323-952d-a8d741c65a68","provider":"VPC","volumeType":"vpc-block","capacity":20,"iops":"3000","tier":null,"az":"eu-gb-1","creationTime":"2019-09-02T16:53:41Z","crn":"crn:v1:bluemix:public:is:eu-gb-1:a/e242f140687cd68a8e037b26680e0f04::volume:3e898aa7-ac71-4323-952d-a8d741c65a68"}]}}`